src/data/transitionzero/TZ-REGI-2024.json

```
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://pathways.rmi.org/schema/pathwayMetadata.v1.json",
+  "id": "TZ-REGI-2024",
+  "publication": {
+    "title": {
+      "full": "From Vision to Voltage"
+    },
+    "subtitle": "Open Access Modelling of the ASEAN Power Grid with TZ-APG",
+    "author": [
+      "Vu, T.",
+      "Shivakumar, A.",
+      "Suarez, I.",
+      "Puspitarini, H. D.",
+      "Majid, A."
+    ],
+    "publisher": {
+      "full": "TransitionZero"
+    },
+    "year": 2024,
+    "month": 12,
+    "day": 2,
+    "links": [
+      {
+        "url": "https://blog.transitionzero.org/hubfs/Analysis/TZ-APG/TZ-APG%20-%20From%20Vision%20to%20Voltage%20-%20Report.pdf",
+        "description": "Report"
+      }
+    ]
+  },
+  "name": { "full": "Regional Integration", "short": "REGI" },
```

corrected in excel source file

```suggestion
  "name": { "full": "Regional Interconnection", "short": "REGI" },
```